recipe/meta.yaml

```diff
@@ -34,3 +34,4 @@ about:
 extra:
   recipe-maintainers:
     - ocefpaf
+    - hobu
```

Comment:
The package for `1.9.3` exists b/c I moved it to broken only until we had a pinning and the `run_exports`. If you try to install, say, `rtree`, you should get `libspatialindex 1.9.3`. 

We should still merge this PR to get you as a maintainer and have the feedstock reflect the available package. Thanks!